kubernetes/reporter/src/main/java/org/arquillian/cube/kubernetes/reporter/TakeKubernetesResourcesInformation.java

```diff
@@ -130,20 +139,53 @@ private TableEntry createTableEntryWithTHead(String name, String... heads) {
         return tableEntry;
     }
 
-    private FileEntry getFileForResourcesConfiguration(Configuration configuration, ReporterConfiguration reporterConfiguration) throws IOException {
-        String path = configuration.getEnvironmentConfigUrl().getPath();
+    private ArrayList<FileEntry> getFilesForResourcesConfiguration(Session session, Configuration configuration, ReporterConfiguration reporterConfiguration) throws IOException {
+        ArrayList<FileEntry> fileEntries = new ArrayList<>();
+        URL environmentConfigUrl = configuration.getEnvironmentConfigUrl();
+
+        if (environmentConfigUrl != null) {
+            fileEntries.add(getFileForResourcesConfiguration(environmentConfigUrl, reporterConfiguration));
+        }
+
+        if (configuration.isEnvironmentInitEnabled()) {
+            List<URL> dependencyUrls = !configuration.getEnvironmentDependencies().isEmpty() ? configuration.getEnvironmentDependencies() : dependencyResolver.get().resolve(session);
```

Comment:
Can `configuration.getEnvironmentDependencies()` be `null`? If so we need to check that too.